tests/test_benchmarks/test_objectnet.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pytest
+
+from brainio.assemblies import BehavioralAssembly
+from brainscore.benchmarks.objectnet import Objectnet
+from brainscore.model_interface import BrainModel
+
+@pytest.mark.private_acces
```

Comment:
@dapello typo, this should be `access` (two s) at the end. I think this is the last thing then all is good